bite-size-notes/week-8-a-Probability Review-Zhijie Tan

```diff
@@ -0,0 +1,204 @@
+
+## Posterior, if Y is a measurement then what is X given Y?
+## Prior, your belief guess of X before knowing Y?
+### Likelihood Function
+ The joint probability mass (or probability density) of observed data is viewed as a function of the parameters of a statistical model.[2]
+- Given X, what you expect Y to be
+- What happens if we have multiple measures?
+- $P(x|y,y_2) \space \propto \space P(y_2|x,y_1)P(x|y_1)$
+
+Assume measurements are independent
+- - $\propto \space P(y_2|x)P(x|y_1)$
+- - $\propto \space P(y_2|x,y_1)P(y_2|x)$   
+- - $\propto \space P(y_2|x,y_1)P(y_1|x)P(x)$ 
+
+## Recursive Bayes Estimates
+A general probabilistic approach for recursively estimating an unknown probability density function (PDF) over time using incoming measurements and a mathematical process model.[2]
+
+Recursive Bayes Estimation Example:
+-- Python Coding
+-- [Recursive Bayes Estimation](https://github.com/ankit-1517/Recursive-Bayesian-Estimation/blob/main/bayes_filter.ipynb)[3]
+
+
+$x$ is (static)
+
+$P(x|y_{1：k})=P(y_k|x)P(x|y_{1:k-1})=\int_x P(y_k|x)P(x|y_{1:k-1})dx$
+
+Posterior from k-1 is your prior for step k
+
+## Recursive Bayesian Rule
+A general probabilistic approach for estimating an unknown probability density function (PDF) recursively over time using incoming measurements and a mathematical process model.[2]
+
+$P(x|y_{1:k}) = \frac{P(y_k|x)P(x|y_{1:k-1})}{\int_{\tilde{x}} P(y_k|\tilde{x})P(\tilde{x}|y_{1:k-1})d\tilde{x}}$
+
+$P(x|y_{1:k-1}) \space \propto \space P(y_{1:k}|x)P(x)$
+
+$P(y_{1:k}|x)=P(y_k|x,y_{1:k-1})P(y_{1:k-1}|x)$
+
+$P(x_1,x_2)=P(x_1:x_2)P(x_2)$ (chian rule)
+
+
+
+Measurements are independently conditioned on X
+
+![](independent.png)
+
+$=P(y_k|x)P(y_{1:k-1}|x)$
+## Transform the formula
+$P(x|y_{1:k}) = {P(y_k|x)P(x|y_{1:k-1})}P(x)$
+
+${P(y_k|x)=P(x|y_{1:k-1})}P(x)$
+
+$P(x|y_{1:k})$ $\propto$ $P(y_{1:k-1}|x)P(x)$ $\rhd {\int_{\tilde{x}} P(y_k|{x})P({x}|y_{1:k-1})d{x}}$
+
+$P(x|y_{1:k})=P(y_k|x)\frac{P(y_{1:k-1}|x)P(x)}{\int_{\tilde{x}} P(y_k|\tilde{x})P(\tilde{x}|y_{1:k-1})d\tilde{x}}=P(x|y_{1:k-1})$
+
+$P(x|y_{1:k})={\int_{\tilde{x}} P(y_k|{x})P({x}|y_{1:k-1})d{x}}\frac{P(y_{1:k-1}|x)P(x)}{\int_{\tilde{x}} P(y_k|\tilde{x})P(\tilde{x}|y_{1:k-1})d\tilde{x}}=\frac{P(y_k|x)P(x|y_{1:k-1})}{\int_{\tilde{x}} P(y_k|\tilde{x})P(\tilde{x}|y_{1:k-1})d\tilde{x}}$
+
+
+
+## Independence 
+$X,Y, RV, X$ is independent of $Y$(Transfor either side) if $P(x,y)=P(x)P(y)$
+
+$\rhd$ $P(x|y)=P(x),P(y|x)=P(y)$
+
+
+## Conditioned independence
+
+$y_1,y_2, conditoned independence X$
+
+$P(y_1|y_2,x)=P(y_1|x)，P(y_2|y_1,x)=P(y_x|x)$
+
+## Recursive Bayesian Filter 
+
+- is change  over time $(x_0,x_1,x_2,..,x_k)$ $P(x_{k+1}|x_{0:k})=P(x_{k+1}|x_k)$
+
+Assume the system is Markov
+- "Next state depends only on the current state"
+- $P(x_0,x_1,x_2,..,x_k)=P(x_0) \pi ^k _{i=1}$P(x_i|x_{i-1})$ Note: no control input
+
+(Markvo Chain Rule) 
+
+![](Markvo.png)
+
+include control input
```

Comment:
"Include control input" should clearly explain the role of control inputs in a Markov Decision Process (MDP).

Reply:
Added some content. Thank you!